web/styles/zulip.css

```diff
@@ -2481,7 +2481,7 @@ select.invite-as {
     }
 }
 
-#custom-expiration-time-unit {
+.custom-input-time-unit {
```

Comment:
This change makes the width of unit dropdown also 220px.

Reply:
Hmm, earlier it was making width `220px` but now after using `modal_select` class there is now change in width.